source/multiple-mongosyncs.txt

```diff
@@ -0,0 +1,281 @@
+
+The ``start`` command options must be the same for all of the ``mongosync``
+instances.
+
+.. _c2c-sharded-progress:
+
+Check Progress
+--------------
+
+The ``mongosync`` instances do not aggregate progress information
+across shards. To review synchronization progress for a particular
+shard, use ``curl`` or another HTTP client to issue the
+:ref:`progress <c2c-api-progress>` command to the ``mongosync``
+instance synchronizing that shard.
+
+.. code-block:: shell
+
+   curl monogsync02Host:27602/api/v1/progress -XGET
+
+This command checks the progress of the ``mongosync`` instance that is
+running on ``monogsync02Host`` and using ``port 27602`` for
+synchronization. To check progress on other shards, update the host and
+port number then repeat the API call to each ``mongosync`` instance.
+
+.. _c2c-sharded-pause:
+
+Pause a ``mongosync`` Instance
+------------------------------
+
+The :ref:`pause <c2c-api-pause>` command will temporarily halt the
+synchronization process on a single shard. It does not pause any other
+``mongosync`` instances that may be running. Use ``curl`` or another
+HTTP client to issue the ``pause`` command to a ``mongosync`` instance.
+
+.. code-block:: shell
+
+   curl monogsync01Host:27601/api/v1/pause -XPOST --data '{}'
+
+This command pauses the ``mongosync`` instance that is running on
+``monogsync01Host`` and using ``port 27601`` for synchronization. To
+pause synchronization on other shards, update the host and port number
+then repeat the API call to each ``mongosync`` instance.
+
+.. _c2c-sharded-resume:
+
+Resume Synchronization
+----------------------
+
+If one or more ``mongosync`` instances are paused, you can use the
+:ref:`resume <c2c-api-resume>` command to resume synchronizing. Run a
+separate ``resume`` command against each paused ``mongosync`` instance
+to continue synchronizing.
+
+Use ``curl`` or another HTTP client to issue the :ref:`resume
+<c2c-api-resume>` command to each ``mongosync`` instance.
+
+.. code-block:: shell
+
+   curl monogsync01Host:27601/api/v1/resume -XPOST --data '{}'
+
+This command resumes synchronization on the ``mongosync`` instance that
+is running on ``monogsync01Host`` and using ``port 27601``.  To 
+resume synchronization on other shards, update the host and port number
+then repeat the API call to each ``mongosync`` instance.
+
+.. _c2c-sharded-commit:
+
+Commit Synchronization From  Multiple ``mongosync`` Instances
+-------------------------------------------------------------
+
+When you want to complete synchronization, issue the :ref:`progress
+<c2c-api-progress>` command. If the ``progress`` command returns,
+``{ canCommit: true }``, issue the :ref:`commit <c2c-api-commit>`
+command. Repeat the process on all of the ``mongosync`` instances.
+
+The ``commit`` operation is blocking. The ``commit`` command will not
+return until ``commit`` has been called on every ``mongosync``
+instance. To avoid unsynchronized data, you should only run the
```

Comment:
With a larger lag, it just mean the commit process would take longer. And the implication for customers is that they will have to block writes on the source for longer while mongosync is trying to drain the lag.

So I wouldn't say "To avoid unsynchronized data", this makes it sound like we could have data consistency problem if we commit with lag.